collections/sentinel-s2-l2a-mosaic-120.yaml

```diff
@@ -56,8 +56,8 @@ BandInformation:
           Notes: DN, typically from 0-10000
         - Name: QM
           Description: QualityMask
-          Notes: DN, 0 - Ok; 1 - no data; 2 - interpolation failed
-Contact:  https://forum.sentinel-hub.com  
+          Notes: DN, from 0-3; OK if larger than 0. More info under the additional info field.
```

Comment:
much better to add the link  to the additional info field
here is the link  `https://collections.sentinel-hub.com/sentinel-s2-l2a-mosaic-120/readme.html`

Reply:
I tried and it seems that the link is not rendered, it just shows it how it is. Should it be done in some other way?

![image](https://user-images.githubusercontent.com/37869250/133075601-2766cbbf-8c90-4cae-b30f-a3823bcd85f5.png)

Perhaps I should try with raw html

Reply:
Yes, raw html works. Fixed.